Python 3.6 support

When using python 3.6 this crashes on a reggex module import.

```
    from dutch_pluralizer import singularize
../../../.pyenv/versions/3.6.9/envs/transformers/lib/python3.6/site-packages/dutch_pluralizer/__init__.py:9: in <module>
    from .pluralizer import pluralize, pluralize_advanced
../../../.pyenv/versions/3.6.9/envs/transformers/lib/python3.6/site-packages/dutch_pluralizer/pluralizer.py:10: in <module>
    from .strategies.pluralize_bastard_words import pluralize_bastard
../../../.pyenv/versions/3.6.9/envs/transformers/lib/python3.6/site-packages/dutch_pluralizer/strategies/pluralize_bastard_words.py:3: in <module>
    from .helpers import VOWELS
../../../.pyenv/versions/3.6.9/envs/transformers/lib/python3.6/site-packages/dutch_pluralizer/strategies/helpers.py:2: in <module>
    from re import Pattern
E   ImportError: cannot import name 'Pattern'
```

--- a/.bumpversion.cfg
+++ b/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.0.37
+current_version = 0.0.38
 commit = True
 tag = True
 

--- a/dutch_pluralizer/__init__.py
+++ b/dutch_pluralizer/__init__.py
@@ -1,4 +1,4 @@
-__version__ = "0.0.37"
+__version__ = "0.0.38"
 
 __all__ = ['pluralize', 'pluralize_advanced', 'singularize',
            'could_be_plural', 'singularize_advanced', 'NounEndingMap', 'Pluralizer']

--- a/dutch_pluralizer/pluralizer.py
+++ b/dutch_pluralizer/pluralizer.py
@@ -7,7 +7,7 @@ from .mapping import NounEndingMap
 from .replacements import (SearchResult, search_by_dictionary,
                            search_by_dictionary_plus_s, search_by_suggestions)
 from .speller import ensure_hunspell_nl, get_plural_nouns
-from .strategies.pluralize_bastard import pluralize_bastard
+from .strategies.pluralize_bastard_words import pluralize_bastard
 from .strategies.pluralize_by_hard_map import pluralize_by_hard_map
 from .strategies.pluralize_by_latin import pluralize_by_latin
 from .strategies.pluralize_eren import pluralize_eren

--- a/dutch_pluralizer/strategies/helpers.py
+++ b/dutch_pluralizer/strategies/helpers.py
@@ -1,9 +1,14 @@
 import re
-from re import Pattern
 from typing import AnyStr
 
+#! only works in Python 3.7+
+# from re import Pattern
+#def create_ends_with_regex(*args: str) -> Pattern:
+#    s = "|".join(map(re.escape, args))
+#    s = f"({s})$"
+#    return re.compile(s)
 
-def create_ends_with_regex(*args: str) -> Pattern:
+def create_ends_with_regex(*args: str):
     s = "|".join(map(re.escape, args))
     s = f"({s})$"
     return re.compile(s)

--- a/publish.ps1
+++ b/publish.ps1
@@ -1,4 +1,4 @@
-$ErrorActionPreference = "Stop"
+    $ErrorActionPreference = "Stop"
 
 $env:PIPENV_VERBOSITY = '-1';
 git config --global core.safecrlf false

--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ README = (HERE / "README.md").read_text()
 # This call to setup() does all the work
 setup(
     name="dutch-pluralizer",
-    version="0.0.37",
+    version="0.0.38",
     description="Generates Dutch plural and singular nouns in a very imperfect way using Hunspell dictionaries. Why imperfect? Because the Dutch language is full of exceptions.",
     long_description=README,
     long_description_content_type="text/markdown",

